chapter2.adoc

```diff
@@ -134,68 +132,147 @@ The privilege level of the trigger chain is determined by the trigger enabled fo
+(draw-box "DXLEN" {:font-size 24 :span 32 :borders {}})
+----
+
+[caption="Register {counter:rimage}: ", reftext="Register {rimage}"]
+[title="Supervisor debug scratch register 0 (sdscratch0)"]
+[id=sdscratch0]
+[bytefield]
+----
+(defattrs :plain [:plain { :font-size 24}])
+(def row-height 40 )
+(def row-header-fn nil)
+(def left-margin 30)
+(def right-margin 30)
+(def boxes-per-row 32)
+(draw-column-headers {:height 24 :font-size 24 :labels (reverse ["0" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "DXLEN-1" ""])})
+(draw-box "sdscratch0" {:span 32:text-anchor "middle" :borders {:left :border-unrelated :top :border-unrelated :bottom :border-unrelated :right :border-unrelated}})
+(draw-box "DXLEN" {:font-size 24 :span 32 :borders {}})
+----
+
+[caption="Register {counter:rimage}: ", reftext="Register {rimage}"]
+[title="Supervisor debug scratch register 1 (sdscratch1)"]
+[id=sdscratch1]
+[bytefield]
+----
+(defattrs :plain [:plain { :font-size 24}])
+(def row-height 40 )
+(def row-header-fn nil)
+(def left-margin 30)
+(def right-margin 30)
+(def boxes-per-row 32)
+(draw-column-headers {:height 24 :font-size 24 :labels (reverse ["0" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "" "DXLEN-1" ""])})
+(draw-box "sdscratch0" {:span 32:text-anchor "middle" :borders {:left :border-unrelated :top :border-unrelated :bottom :border-unrelated :right :border-unrelated}})
+(draw-box "DXLEN" {:font-size 24 :span 32 :borders {}})
+----
+
+==== Extension of Sdtrig CSR
+
+The Smtdeleg and Sstcfg extensions define the process for delegating triggers to modes with lower privilege than M-mode. The Sdsec requires both extensions to securely delegate Sdtrig triggers to supervisor domain.
 
-Debug PC (at 0x7b1) and Debug Scratch Register (at 0x7b2 and 0x7b3) are not restricted by <<dbgaccpriv, debug access privilege>>, they are always accessible in Debug Mode.
+[NOTE]
+When M-mode enables debugging for supervisor domain, it can optionally delegate the triggers to the supervisor domain, allowing an external debugger with S-mode privilege to configure these triggers.
 
 [[Sdseccsr]]
-==== Sdsec CSR
+==== Debug Control CSR
 
-The Sdsec extension does not introduce any new CSR. The CSR control knobs in `msdcfg` for supervisor domain debug and trace are specified in Smsdedbg and Smsdetrc extension respectively in _RISC-V Supervisor Domains Access Protection_ cite:[smmtt]. The Smsdedbg and/or Smsdetrc extension must be implemented to support security control for debugging and/or tracing in supervisor domain.
+The CSR holding  the debug and trace contol knobs for supervisor domain are specified in Smsdedbg and Smsdetrc extension respectively in _RISC-V Supervisor Domains Access Protection_ cite:[smmtt]. The Smsdedbg and/or Smsdetrc extension must be implemented to support security control for debugging and/or tracing in supervisor domain.
```

Comment:
Should CSR be plural above?

Reply:
Both `sdetrcalw` and `adedbgalw` are located within  msdcfg CSR, thus I didnt use plural form.